src/web/styles.scss

```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+
+  & th,
```

Comment:
On second look, actually the `&` is not at all necessary here because without it the meaning is still the same.